WebCryptoAPI/wrapKey_unwrapKey/wrapKey_unwrapKey.https.any.js

```diff
@@ -9,7 +11,8 @@ function run_test() {
 
     // Generate all the keys needed, then iterate over all combinations
     // to test wrapping and unwrapping.
-    Promise.all([generateWrappingKeys(), generateKeysToWrap(), generateEcdhPeerKey()])
+    promise_test(function() {
+    return Promise.all([generateWrappingKeys(), generateKeysToWrap(), generateEcdhPeerKey()])
```

Comment:
I think the error handling below can probably go away now?

Reply:
Removed.